sketch/tensor_slide_flat.hpp

```diff
@@ -0,0 +1,45 @@
+#include "sketch/tensor_slide.hpp"
+
+namespace ts {
+
+// Flattener: one of the classes from util/dim_reduce.h.
```

Comment:
Can you make this a proper doxygen and also add a quick description of the class. (this code is likely to be published with the project)